src/cs/ui/ImportSlider.cs

```diff
@@ -119,5 +120,11 @@ public partial class ImportSlider : HSlider {
 		// Hide all buttons
 		ApplySelection.Hide();
 		Cancel.Hide();
+		this.Hide();
+	}
+	
+	private void _on_imports_b_pressed() {
+	this.Visible = !this.Visible;
```

Comment:
Same comment as above, please use c# naming conventions and connect the signals in code.